packages/plugin-moxie-limit-order/src/utils/0xApis.ts

```diff
@@ -4,6 +4,7 @@ import { createClientV2 } from "@0x/swap-ts-sdk";
 import { elizaLogger } from "@moxie-protocol/core";
 import { ethers } from "ethers";
 import { ERC20_TXN_SLIPPAGE_BPS } from "../constants";
+import { mockGetQuoteResponse } from "../constants/constants";
 
 const initializeClients = () => {
     if (!process.env.ZERO_EX_API_KEY) {
```

Comment:
You'll also need to remove these checks that will revert if no API key is added by user

Reply:
added log instead of error